python/Ganga/Lib/LCG/Grid.py

```diff
@@ -1,9 +1,9 @@
 import os
 import re
 import tempfile
-import time
+import datetime
 
-from Ganga.GPIDev.Credentials import getCredential
+from Ganga.GPIDev.Credentials2 import credential_store
```

Comment:
Indeed. Unfortunately while this PR is being finalised, there is still some ATLAS code which depends on the old credentials system. Once that is ported, `Credentials` can be removed and `Credentials2` can be renamed.